doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -121,7 +121,7 @@ by the coercions ``f₁..fₖ``.  The application of a coercion path to a
 term consists of the successive application of its coercions.
 
 
-Declaration of Coercions
+Declaring of Coercions
```

Comment:
It seems to me that `of` is not needed here: "Declaring Coercions"

Reply:
Right, trying to make it sound better I introduced even more errors : )